src/components/dashboard/SendLinkSummary.js

```diff
@@ -54,13 +54,21 @@ const SendLinkSummary = ({ screenProps }: AmountProps) => {
             e.message
           }`,
           dismissText: 'Ok',
-          onDismiss: () =>
-            screenProps.push('SendConfirmation', {
+          onDismiss: () => {
+            const desktopShareLink = generateSendShareText(
               paymentLink,
               amount,
+              counterPartyDisplayName,
+              profile.fullName
+            )
+
+            screenProps.push('SendConfirmation', {
+              paymentLink: desktopShareLink,
```

Comment:
it seems paymentLink  is not used in confirmation, you generate it but i couldnt find any place where you added usage.

Reply:
@sirpy it is used in sendConfirmation screen. PaymentLink content will be copied to buffer.